src/main/java/com/epam/ta/reportportal/ws/converter/builders/TestItemBuilder.java

```diff
@@ -152,7 +154,7 @@ public TestItemBuilder addParameters(List<ParameterResource> parameters) {
 			testItem.setParameters(parameters.stream().map(it -> {
 				Parameter parameter = new Parameter();
 				parameter.setKey(it.getKey());
-				parameter.setValue(it.getValue());
+				parameter.setValue(ofNullable(it.getValue()).orElse(PARAMETER_NULL_VALUE));
```

Comment:
i think that it's not expected behaviour, so it should be validated somewhere on the request level